sky/utils/controller_utils.py

```diff
@@ -350,8 +351,19 @@ def download_and_stream_latest_job_log(
 
 
 def shared_controller_vars_to_fill(
-        controller: Controllers,
-        remote_user_config_path: str) -> Dict[str, str]:
+        controller: Controllers, remote_user_config_path: str,
+        local_user_config: Dict[str, Any]) -> Dict[str, str]:
+    if not local_user_config:
+        local_user_config_path = None
+    else:
+        local_user_config.pop('admin_policy', None)
```

Comment:
Does this mean the policy is executed on client side only, never on serve/job controller? Worth a comment.

Reply:
Good point! Added